packages/cli/test/integration/generators/copyright.integration.js

```diff
@@ -83,6 +83,41 @@ describe('lb4 copyright', /** @this {Mocha.Suite} */ function () {
     );
   });
 
+  it('updates custom copyright/license headers with options', async () => {
+    await testUtils
+      .executeGenerator(generator)
+      .inDir(sandbox.path, () =>
+        testUtils.givenLBProject(sandbox.path, {
+          excludePackageJSON: true,
+          additionalFiles: SANDBOX_FILES,
+        }),
+      )
+      .withOptions({
+        owner: 'ACME Inc.',
+        license: 'CUSTOM',
+        gitOnly: false,
+      })
+      .withPrompts({
+        customLicenseLines: `
+=============================================================================
+Licensed Materials - Property of <%= owner %>
+(C) Copyright <%= owner %> <%= years %>
+US Government Users Restricted Rights - Use, duplication or disclosure
+restricted by GSA ADP Schedule Contract with <%= owner %>.
+=============================================================================`,
+      });
+
+    assertHeader(
+      ['src/application.ts', 'lib/no-header.js'],
+      `=============================================================================`,
+      `Licensed Materials - Property of ACME Inc.`,
+      `(C) Copyright ACME Inc. ${year}`,
```

Comment:
Nitpick: can you add the year too?

Reply:
I'm not sure what you referred to. The `${year}` is replaced by JS for its value.